.devcontainer.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "Go for aks-engine",
+	"image": "deis/go-dev",
```

Comment:
We should use the same image as `make dev` and our CI pipeline do: `quay.io/deis/go-dev:v1.22.3`. It's a pain to have to update the tag, but it avoids surprises.